lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -677,15 +677,20 @@ def install_a_puppet_msi_on(hosts, opts)
 
             if host.is_cygwin?
               # NOTE: it is critical that -o be before -O on Windows
-              on host, "curl -o \"#{msi_download_path}\" -O #{link}"
+              proxy = opts[:package_proxy] ? " -x #{opts[:package_proxy]}" : ''
+              on host, "curl --connect-timeout 5 --retry 20 --retry-delay 10#{proxy} -o \"#{msi_download_path}\" -O #{link}"
```

Comment:
we should add only the proxy option, without hardcoded `--connect-timeout 5 --retry 20 --retry-delay 10`

Reply:
Sure, i'll make the adjustment now.

Reply:
Done.